src/index.html

```diff
@@ -12,7 +12,40 @@
       href="styles/index.scss"
```

Comment:
Stylesheets with a .scss extension should not be linked directly in HTML. You should compile them to CSS and then link the .css file.